docs/admin/networking/index.md

```diff
@@ -123,8 +123,7 @@ but this can be disabled or changed for
 By default, your Coder server also runs a built-in DERP relay which can be used
 for both public and [Air-gapped deployments](../../install/airgap.md).
 
-However, our Wireguard integration through Tailscale has graciously allowed us
-to use
+However, Tailscale has graciously allowed us to use
```

Comment:
Graciously feels out of place, but maybe that's me. Perhaps
```suggestion
Coder also supports Tailscale's global DERP relays
```
You'll need to fix the link and wording

Reply:
@david-fraley "graciously" is used here because Tailscale's DERP relays aren't really meant to be used by products like Coder. They are maintained by Tailscale to be used with Tailscale, not with products that forked Tailscale for their own purposes, like Coder.

In the docs, we are basically saying that Coder users may piggyback on Tailscale's infrastructure, that we asked Tailscale if that's ok, and that they graciously agreed.

It doesn't seem apparent from how we phrased this. Do you have any ideas for how we could rephrase it to avoid further confusion?

Reply:
```suggestion
Tailscale maintains a global fleet of DERP relays intended for their product, and has agreed to allow Coder to access and use them. 
```